views/case-building/reasons-for-appeal/decision-appeal.njk

```diff
@@ -0,0 +1,44 @@
+{% set pageTitleName = i18n.pages.wrongHomeOfficeDecision.title %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% extends "layout.njk" %}
+
+
+{% block content %}
+    <h1 class="govuk-fieldset__legend--xl">{{ i18n.pages.wrongHomeOfficeDecision.heading | safe }}</h1>
+    <p class="govuk-body">{{ i18n.pages.wrongHomeOfficeDecision.usefulInfo |  safe  }}</p>
+    {% if errorList %}
+        {{ govukErrorSummary({
+            titleText: i18n.error.summary.title,
+            descriptionText: i18n.error.summary.descriptionText,
+            errorList: errorList
+        })
+        }}
+    {% endif %}
+    <form type="submit"  action="/reasons/decision-appeal" method="post">
+    {{ govukTextarea({
+        name: "moreDetail",
+        id: "moreDetail",
+        label: {
+            text: i18n.pages.wrongHomeOfficeDecision.textTitle | safe
+        },
+        errorMessage: {
+            text:  error['moreDetail'].text
+        } if error['moreDetail']
+    }) }}
+    <div class="panel-background">
+        <ul>
+            <li class="list-item-no-bullet"><h4 class="govuk-heading-s">{{ i18n.pages.wrongHomeOfficeDecision.howToAnswer }}</h4></li>
+            <li>{{ i18n.pages.wrongHomeOfficeDecision.bulletOne | safe}}</li>
+            <li>{{ i18n.pages.wrongHomeOfficeDecision.bulletTwo | safe}}</li>
+            <li>{{ i18n.pages.wrongHomeOfficeDecision.bulletThree | safe }}</li>
+            <li>{{ i18n.pages.wrongHomeOfficeDecision.bulletFour | safe }}</li>
+            <li>{{ i18n.pages.wrongHomeOfficeDecision.bulletFive | safe}}</li>
+            <br/>
+            <li class="list-item-no-bullet"><h4 class="govuk-heading-s">{{ i18n.pages.wrongHomeOfficeDecision.needHelp }}</h4></li>
+            <li><a href="#">{{ i18n.pages.wrongHomeOfficeDecision.bulletLink }}</a></li>
+        </ul>
```

Comment:
<li> usually lists dont contains headers, instead put the title before the list tag and split it in two differents lists...
something like:

`<h4>...</h4>`
`<ul> ... </ul>`

`<h4>...</h4>`
`<ul> ... </ul>`